docs/src/api/class-request.md

```diff
@@ -183,6 +183,17 @@ following: `document`, `stylesheet`, `image`, `media`, `font`, `script`, `texttr
 
 Returns the matching [Response] object, or `null` if the response was not received due to error.
 
+## method: Request.sizes
+- returns: <[Object]>
+  - `requestBodySize` <[int]> Size of the request body (POST data payload) in bytes. Set to 0 if there was no body.
+  - `requestHeadersSize` <[float]> Total number of bytes from the start of the HTTP request message until (and including) the double CRLF before the body.
+  - `responseBodySize` <[int]> Size of the received response body in bytes.
```

Comment:
Do we have a good idea on why users could want original body size? And why they don't want the actual transfer size for the request body?

Reply:
It's from the HAR spec and I thought it's about exposing these values.

Reply:
You should look at har spec and re-read my question!